client/src/components/Register.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react'
+import './Register.css'
+
+const Register = () => {
+    return (
+        <div className='Register'>
```

Comment:
Nit: lowercase on 'Register' for consistency

Reply:
Changed it!